front_end/entrypoints/inspector_main/BUILD.gn

```diff
@@ -22,6 +23,7 @@ devtools_module("inspector_main") {
     "../../panels/mobile_throttling:bundle",
     "../../ui/legacy:bundle",
     "../../ui/legacy/components/utils:bundle",
+    "../../ui/components/buttons:bundle",
```

Comment:
This is smelly. There are buttons in the default tabs e.g. the "Memory" tab has a `Take heap snapshot` button, but buttons aren't built with the `inspector_main`. Are there other buttons, or am I missing something?

Reply:
this is the button used in the `CohtmlPanel`. It is used in the typescript and when the inspector is built, the `button.js` is not in the final bundle. When HTTP server is compiled it then has no `button.js` but the file is requested when our inspector is opened in Chrome